.tav.yml

```diff
@@ -476,25 +488,32 @@ fastify-v2:
 fastify-v3:
   name: fastify
   # Update versions periodically with "./dev-utils/tav-versions-fastify.sh".
-  versions: '3.0.0 || 3.6.0 || 3.14.2 || 3.20.1 || 3.24.0 || 3.29.0 || 3.29.5 || >3.29.5 <4' # subset of '>=3 <4'
+  versions: '3.0.0 || 3.6.0 || 3.14.2 || 3.20.1 || 3.24.0 || 3.29.0 || ^3.29.5'
   peerDependencies: '@fastify/formbody@^6.0.1'
   node: '>=10'
   commands:
     - node test/instrumentation/modules/fastify/fastify.test.js
     - node test/instrumentation/modules/fastify/async-await.test.js
     - node test/instrumentation/modules/fastify/set-framework.test.js
     - node test/sanitize-field-names/fastify.test.js
+  update-versions:
+    mode: max-5
+    range: '>=3 <4'
 fastify:
   name: fastify
   # Update versions periodically with "./dev-utils/tav-versions-fastify.sh".
-  versions: '4.0.0 || 4.2.1 || 4.5.2 || 4.8.1 || 4.10.1 || 4.14.0 || 4.17.0 || >4.17.0' # subset of '>=4 <4.0.1 || >4.0.1 <4.16.0 || >4.16.2'
+  versions: '4.0.0 || 4.5.0 || 4.9.1 || 4.14.1 || 4.19.1 || 4.23.2 || ^4.24.3'
+  versions-stash: '4.0.0 || 4.2.1 || 4.5.2 || 4.8.1 || 4.10.1 || 4.14.0 || 4.17.0 || >4.17.0' # subset of '>=4 <4.0.1 || >4.0.1 <4.16.0 || >4.16.2'
```

Comment:
Note for reviewer: this range here is a bit tricky and current implementation does not support it. I think it can be done by enriching the `update-versions` config. Instead of having a `range` configuration we should split in two configs:
- `include` (required): works exactly as `range`
- `exclude` (optional): provides a range of versions we want to filter out

In this case the range `'>=4 <4.0.1 || >4.0.1 <4.16.0 || >4.16.2'` can be translated to

```
- include: '>=4 <5'
- exclude: '4.0.1 || >=4.16.0 <=4.6.2'
```